package.json

```diff
@@ -344,136 +347,56 @@
-                    "group": "regs_reg_2_destructive@1"
-                },
-                {
-                    "command": "vscode-docker.registries.private.disconnectRegistry",
-                    "when": "view == dockerRegistries && viewItem == privateRegistry",
+                    "when": "view == dockerRegistries && viewItem == azure;DockerV2;Registry;",
                     "group": "regs_reg_2_destructive@1"
                 },
-                {
-                    "command": "vscode-docker.registries.azure.openInPortal",
-                    "when": "view == dockerRegistries && viewItem == azureRegistry",
-                    "group": "regs_reg_9_common@1"
-                },
-                {
-                    "command": "vscode-docker.registries.dockerHub.openInBrowser",
-                    "when": "view == dockerRegistries && viewItem == dockerHubRegistry",
-                    "group": "regs_reg_9_common@1"
-                },
-                {
-                    "command": "vscode-docker.registries.azure.viewProperties",
-                    "when": "view == dockerRegistries && viewItem == azureRegistry",
-                    "group": "regs_reg_9_common@2"
-                },
-                {
-                    "command": "vscode-docker.registries.refresh",
-                    "when": "view == dockerRegistries && viewItem =~ /registry$/i",
-                    "group": "regs_reg_9_common@9"
-                },
                 {
                     "command": "vscode-docker.registries.pullRepository",
-                    "when": "view == dockerRegistries && viewItem =~ /repository$/i",
+                    "when": "view == dockerRegistries && viewItem =~ /Repository;/",
                     "group": "regs_repo_1_general@1"
                 },
                 {
                     "command": "vscode-docker.registries.azure.deleteRepository",
-                    "when": "view == dockerRegistries && viewItem == azureRepository",
+                    "when": "view == dockerRegistries && viewItem == azure;DockerV2;Repository;",
                     "group": "regs_repo_2_destructive@1"
                 },
-                {
-                    "command": "vscode-docker.registries.dockerHub.openInBrowser",
-                    "when": "view == dockerRegistries && viewItem == dockerHubRepository",
-                    "group": "regs_repo_9_common@1"
-                },
-                {
-                    "command": "vscode-docker.registries.refresh",
-                    "when": "view == dockerRegistries && viewItem =~ /repository$/i",
-                    "group": "regs_repo_9_common@9"
-                },
                 {
                     "command": "vscode-docker.registries.pullImage",
-                    "when": "view == dockerRegistries && viewItem =~ /tag$/i",
+                    "when": "view == dockerRegistries && viewItem =~ /Tag;/",
```

Comment:
Pardon my Dockgerance (Docker + ignorance), but why are all the viewItems for the images called Tags?

Reply:
It's confusing naming for sure and there's no official consensus. Take this as an example:
<img width="229" alt="Screen Shot 2019-07-02 at 4 37 37 PM" src="https://user-images.githubusercontent.com/11282622/60553338-b932bf80-9ce7-11e9-9f4b-45dd62bdbea4.png">

Generally, `latest` and `v3` are considered tags, `repo1` is considered a repository and `repo1:latest` and `repo1:v3` are considered images.